src/vm/readytoruninfo.cpp

```diff
@@ -497,13 +497,15 @@ PTR_ReadyToRunInfo ReadyToRunInfo::Initialize(Module * pModule, AllocMemTracker
     }
 
 #ifdef FEATURE_NATIVE_IMAGE_GENERATION
+#ifndef _TARGET_ARM_
```

Comment:
I do not think that this is a good fix. It may cause invalid native images to be produced when the reference assemblies passed to NGen are compiled with R2R.